DevIncubator_Autopark/AllCollections/MyQueue.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace DevIncubator_Autopark.AllCollections
+{
+    public class MyQueue<T> : IEnumerable<T>
+    {
+        private const int DefaultCapacity = 10;
+        private const int ResizeValue = 2;
+
+        public int Count { get => _endIndex; }
+
+        private T[] _queue;
+        private int _endIndex;
+
+        public MyQueue()
+        {
+            _queue = new T[DefaultCapacity];
+
+            _endIndex = 0;
+        }
+
+        public MyQueue(T[] queue)
+        {
+            if (queue != null)
```

Comment:
```suggestion
            if (queue is null)
```
typo `queue != null`

Reply:
Fixed